src/components/SingleReviewPage.jsx

```diff
@@ -0,0 +1,67 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import SingleReviewCard from "./SingleReviewCard";
+import ErrorComponent from "./ErrorComponent";
+// import CommentCard from "./CommentCard";
+
+const SingleReviewPage = () => {
+  const { review_id } = useParams();
+  const [review, setReview] = useState(null);
+  // const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    setIsLoading(true);
+    fetch(`https://ncgamesapp.herokuapp.com/api/reviews/${review_id}`)
+      .then((res) => {
+        if (res.ok) {
+          return res.json();
+        }
+        throw Error(`${res.status}: ${res.statusText}`);
+      })
+      .then(({ review }) => {
+        setReview(review);
```

Comment:
we dont set the error back to null here, this could cause some problems if we can navigate between reviews :)